fabric-transfer-api-v1/src/main/java/net/fabricmc/fabric/api/transfer/v1/storage/Movement.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.transfer.v1.storage;
+
+import java.util.Iterator;
+import java.util.function.Predicate;
+
+import org.jetbrains.annotations.Nullable;
+
+import net.fabricmc.fabric.api.transfer.v1.transaction.Transaction;
+
+/**
+ * Helper functions to move resources between two {@link Storage}s.
+ */
+public final class Movement {
```

Comment:
perhaps a `move` function that takes a "modulo" so for example if you wanted to move only in 1/3, it would try once, try again with the "lower value" (to account for capacity) and then fail

Reply:
Not sure that's useful tbh.

Reply:
Marking as resolved - I don't think it is.